src/languages/index.ts

```diff
@@ -19,7 +19,6 @@ export const resources: {
   [key in Language]: Translation;
 } = {
   en: english,
-  // @ts-ignore
```

Comment:
Key change here. Our translation objects are guaranteed by TS to be the same shape.